django_app/redbox_app/redbox_core/views.py

```diff
@@ -11,6 +11,9 @@
 from django.urls import reverse
 from django.utils.datastructures import MultiValueDictKeyError
 from django.views.decorators.http import require_http_methods
+from requests.exceptions import HTTPError
```

Comment:
Are you still moving the imports around? You might need to either fix your editor, or run `make format lint` before committing.

Reply:
I think it's possibly because I have the django_app open in a separate pycharm window so pycharm doesn't respect the root pyproject.toml linting rules

Reply:
Probably, yes.